docs/querying/sql-window-functions.md

```diff
@@ -0,0 +1,221 @@
+---
+id: sql-window-functions
+title: Window functions
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ License); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ AS IS BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+:::info
+
+Apache Druid supports two query languages: [Druid SQL](sql.md) and [native queries](querying.md).
+This document describes the SQL language.
```

Comment:
I'm not sure if we need these lines ; I believe this file will open from `SQL` sidebard stuff; so I feel they are  pretty redundant

Reply:
This is in all the SQL reference docs. This admonition is here for all the SQL docs, so it should be in this one too. We can consider removing as an effort to remove from all SQL